guides/source/active_record_querying.md

```diff
@@ -2162,6 +2162,57 @@ NOTE: Note that if a query matches multiple records, `find_by` will
 fetch only the first one and ignore the others (see the `LIMIT 1`
 statement above).
 
+### Using Common Table Expression (CTE)
+
+The [`with`][] method is a convenient way to access [Common Table Expressions (CTE)](https://en.wikipedia.org/wiki/Hierarchical_and_recursive_queries_in_SQL#Common_table_expression)'s WITH syntax, which can define named temporal query results that are durable and referable within a single SQL query.
```

Comment:
```suggestion
The [`with`][] method is a convenient way to access [Common Table Expressions (CTE)](https://en.wikipedia.org/wiki/Hierarchical_and_recursive_queries_in_SQL#Common_table_expression) `WITH` syntax, which can define named temporal query results that are referable within a single SQL query.
```

"define named temporal query results " is still a hard statement to deconstruct, but I think this is reasonable